supervisord.ini

```diff
@@ -1,13 +1,24 @@
 [supervisord]
 nodaemon=true
 
+[supervisorctl]
+serverurl=unix:///run/supervisord.sock
+
+[unix_http_server]
+file = /run/supervisord.sock
+chmod = 0777
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
```

Comment:
I know that we should stop using supervisor and such deploying multiple services inside one pod... but in a meantime I wanted to make `supervisorctl` command work:
```
supervisorctl status
grafana                          RUNNING   pid 16, uptime 0:00:16
prometheus                       RUNNING   pid 17, uptime 0:00:16
```